trunk/projects/h5fs_test/test_readimage.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
```

Comment:
Without the unsorted.hdf5 file this application is not very useful. Also logic changes above may make the factory implementation go away. I prefer this type of sanity check to go in jenkins for testing when we build a new AMI and test the library for functionality.